src/content/changes/highlights/2021-12-03.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2021-12-03T10:00:00
+title: Highlights for the week ending December 03, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.5.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#250---2021-11-29) updates the controller image to upstream [v1.1.0](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.1.0). [v2.6.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#260---2021-12-02) adds the ability to specify a custom LoadBalancer Service name suffix.
+
+
+## User interfaces
+ - Version caching is now disabled for the selfupdate command, so that the latest version can be seen right after it’s released.
+ - Make the --release flag mandatory, which specifies the workload cluster release version in the template cluster and template nodepool subcommands.
```

Comment:
not sure if this is important change to include, but if you think so, please also specify that this is a flag for `kubectl gs` :)

Reply:
kubectl gs added